www/js/con-author.js

```diff
@@ -588,6 +588,14 @@ define([
 			var logObj = {};
 			if(model && model == "authored"){
 				var directives = [];
+				//before parsing the equation we need to check if all variables are present and none of the variable fields is left empty
```

Comment:
Looks good, but I found a bug related to this function--if the user clicks "Delete Node" this check still triggers.  The equation done handler shouldn't need to be called on delete I think...